customization-tests/src/main/java/fixtures/bodycomplex/implementation/ArraysImpl.java

```diff
@@ -50,33 +51,38 @@
         @Get("/complex/array/valid")
         @ExpectedResponses({200})
         @UnexpectedResponseExceptionType(ErrorException.class)
-        Mono<Response<ArrayWrapper>> getValid(@HostParam("$host") String host, Context context);
+        Mono<Response<ArrayWrapper>> getValid(
+                @HostParam("$host") String host, @HeaderParam("Accept") String accept, Context context);
 
         @Put("/complex/array/valid")
         @ExpectedResponses({200})
         @UnexpectedResponseExceptionType(ErrorException.class)
         Mono<Response<Void>> putValid(
                 @HostParam("$host") String host,
                 @BodyParam("application/json") ArrayWrapper complexBody,
+                @HeaderParam("Accept") String accept,
                 Context context);
 
         @Get("/complex/array/empty")
         @ExpectedResponses({200})
         @UnexpectedResponseExceptionType(ErrorException.class)
-        Mono<Response<ArrayWrapper>> getEmpty(@HostParam("$host") String host, Context context);
+        Mono<Response<ArrayWrapper>> getEmpty(
+                @HostParam("$host") String host, @HeaderParam("Accept") String accept, Context context);
 
         @Put("/complex/array/empty")
         @ExpectedResponses({200})
         @UnexpectedResponseExceptionType(ErrorException.class)
         Mono<Response<Void>> putEmpty(
                 @HostParam("$host") String host,
                 @BodyParam("application/json") ArrayWrapper complexBody,
+                @HeaderParam("Accept") String accept,
                 Context context);
 
         @Get("/complex/array/notprovided")
         @ExpectedResponses({200})
         @UnexpectedResponseExceptionType(ErrorException.class)
-        Mono<Response<ArrayWrapper>> getNotProvided(@HostParam("$host") String host, Context context);
+        Mono<Response<ArrayWrapper>> getNotProvided(
+                @HostParam("$host") String host, @HeaderParam("Accept") String accept, Context context);
     }
```

Comment:
Let me know whether this change will affect data-plane.

m4 added feature to fill-in `Accept` header, probably based on `produces` field in swagger.